files/chef-marketplace-cookbooks/chef-marketplace/recipes/_automate_enable.rb

```diff
@@ -20,9 +21,14 @@
   type              :rsa
 end
 
-# TODO:
-# * reconfigure delivery when templates change
+# Use a 30 day trial license if we're on Azure
```

Comment:
How does the 30 day trial license work? Does it start at the time of activation?